docs/source/quickstart.rst

```diff
@@ -119,31 +133,31 @@ We can send requests to the newly started service with any clients.
         requests.post(
             "http://127.0.0.1:5000/predict",
             headers={"content-type": "application/json"},
-            data="[[5,4,3,2]]").text
+            data="[5,4,3,2]").text
 
     .. code-tab:: bash
 
         > curl \
```

Comment:
This should be in oneline so that the sphinx doesn't format the escaped space in the rendered code block

Reply:
I don't see what you mean; the code block looks fine?